_posts/2021-12-28-trino-2021-a-year-of-growth.md

```diff
@@ -0,0 +1,356 @@
+
+### HDFS, S3, Azure and cloud object storage systems
+
+* Fixed queries on empty Iceberg tables without a snapshot ID that were created by Spark.
+* Updated to Iceberg 0.11.0 behavior for transforms of dates and timestamps before 1970.
+* Added procedure `system.flush_metadata_cache()` to flush metadata caches.
+* Improved Hive performance by not generating splits for empty files.
+* Improved Iceberg query performance when dynamic filtering can be leveraged.
+* Improved Iceberg performance when reading timestamps from Parquet files.
+* Improved Iceberg performance for queries on nested data through dereference pushdown.
+* Added support for insert overwrite operations on S3-backed tables.
+* Added support for Iceberg `uuid` type.
+* Added Iceberg support for Trino views.
+
+
+### Elasticsearch connector
+
+* Added support for reading fields as `json` values.
+* Fixed failure when documents contain fields of unsupported types.
+* Added support for `scaled_float` type.
+* Added support for assuming an IAM role.
+* Fixed potential incorrect results when queries contain an `IS NULL` predicate.
+* Added retry requests with backoff when Elasticsearch is overloaded.
+
+### Kafka connector
+
+* Added support for `TLS` security protocol.
+* Fixed querying of Schema Registry tables with References in their schema.
+* Fixed listing of schema registry tables having ambiguous subject name in lower case.
+
+### MemSQL connector
+
+* Allowed limiting the size of the metadata cache via the 
+`metadata.cache-maximum-size` configuration property.
+* Added partial support for `DELETE` statement where predicate can be fully pushed 
+down to the remote datasource.
+* Added support for MemSQL 3.2.
+
+### MongoDB connector
+
+* Added [`timestamp_objectid()`]({{site.url}}/docs/current/connector/mongodb.html#timestamp_objectid)
+function.
+* Enabled `mongodb.socket-keep-alive` config property by default.
+* Add support for `json` type.
+* Support reading MongoDB `DBRef` type.
+* Allow skipping creation of an index for the `_schema` collection if it already
+exists.
+* Added support to redact the value of `mongodb.credentials` in the server log.
+* Added support for dropping columns.
+
+### MySQL connector
+
+* Added support for reading and writing `timestamp` values with precision higher than 3.
+* Added support for predicate pushdown on `timestamp` columns.
+* Exclude an internal `sys` schema from schema listings.
+
+### Pinot connector
+
+* Updated Pinot connector to be compatible with versions >= 0.8.0 and drop 
+support for older versions.
+* Added support for pushdown of filters on `varbinary` columns to Pinot.
+* Fixed incorrect results for queries that contain aggregations and `IN` and 
+`NOT IN` filters over varchar columns.
+* Fixed failure for queries with filters on `real` or `double` columns having 
+`+Infinity` or `-Infinity` values.
+* Implemented aggregation pushdown for `count`, `avg`, `min`, `max`, `sum`, 
```

Comment:
Just use 

Implemented aggregation pushdown.


and get rid of the rest .. this is just part of the documentation..